server/game/cards/04-MM/Ant110ny.js

```diff
@@ -0,0 +1,26 @@
+const Card = require('../../Card.js');
+
+class Ant110ny extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            gameAction: ability.actions.capture(context => ({
+                amount: context.player.opponent.amber
+            }))
+        });
+        this.interrupt({
+            when: {
+                onRoundEnded: (event, context) =>
+                    context.player === this.game.activePlayer
+            },
+            condition: context => context.source.hasToken('amber'),
+            gameAction: [
+                ability.actions.removeAmber({ amount: 1 }),
+                ability.actions.gainAmber(context => ({ target: context.player.opponent }))
```

Comment:
You might want to add a test in conjunction with Exile, but I think considering other cards like Shaffle and Scowly Capper, it will sorry!

Reply:
added test and added what you told me to add master